cluster/addons/fluentd-elasticsearch/es-image/run.sh

```diff
@@ -22,4 +22,6 @@ export MINIMUM_MASTER_NODES=${MINIMUM_MASTER_NODES:-2}
 
 /elasticsearch_logging_discovery >> /elasticsearch/config/elasticsearch.yml
 
-/elasticsearch/bin/elasticsearch
+chown -R elasticsearch:elasticsearch /data
```

Comment:
Elasticsearch doesn't work as root in the newer version, so we had to introduced privilege dropping, the question is at what point

In this change, `run.sh` started working with root privileges, so at this point we will have enough permissions to perform a chown, but elasticsearch will be executed as `elasticsearch` user later